components/61-layouts/detail-layout/_detail-layout.scss

```diff
@@ -206,6 +206,10 @@ aside.bottom {
 
     @include tablet {
       margin-bottom: 120px;
+
+      .node-type-product & {
```

Comment:
Geen content types toevoegen aan Fractal (wat platform- en projectonafhankelijk is)